src/index.js

```diff
@@ -5,6 +5,16 @@ module.exports = {
     const { PixelbinConfig, PixelbinClient } = require("@pixelbin/admin");
     const pixelbinConfig = new PixelbinConfig(config);
     const pixelbinClient = new PixelbinClient(pixelbinConfig);
+    const defaultPath = config.folderName || "strapi-images";
```

Comment:
Change variable name from `defaultPath` to `folderPath` or something similar

make `DEFAULT_PATH` as constant and value to be `"strapi-images"`